config/initializers/redcarpet.rb

```diff
@@ -0,0 +1,3 @@
+require 'redcarpet'
+
+Rails.configuration.redcarpet = Redcarpet::Markdown.new(Redcarpet::Render::HTML.new)
```

Comment:
This is a simple way of creating a _single_ instance of Redcarpet and reuse it everywhere else, instead of creating new objects every time we want to render Markdown into HTML. Happy to hear alternatives! 🙂